iati/about/models.py

```diff
@@ -1,8 +1,150 @@
+    ]
+
+
+class AboutSubPageDocument(Orderable):
+    """A model for About sub page documents."""
+
+    page = ParentalKey(AboutSubPage, related_name='about_sub_page_documents')
+    document = models.ForeignKey(
+        'wagtaildocs.Document',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
+
+    panels = [
+        DocumentChooserPanel('document'),
+    ]
+
+
+class CaseStudyIndexPage(AbstractIndexPage):
+    """A model for the Case Studies Index page."""
+
+    subpage_types = ['about.CaseStudyPage']
+
+    @property
+    def case_studies(self):
+        """Get all CaseStudyPage objects that have been published."""
+        case_studies = CaseStudyPage.objects.child_of(self).live()
+        return case_studies
+
+    def get_context(self, request):
+        """Overwrite the default wagtail get_context function to allow for pagination.
+
+        Use the functions built into the abstract index page class to apply pagination, limiting the results to 3 per page.
+
+        """
+        children = self.case_studies
+        paginated_children = self.paginate(request, children, max_results=3)
+        context = super(CaseStudyIndexPage, self).get_context(request)
+        context['case_studies'] = paginated_children
+        return context
+
+
+class CaseStudyPage(AbstractContentPage):
+    """A model for Case Study pages."""
+
+    parent_page_types = ['about.CaseStudyIndexPage']
+    subpage_types = []
+
+    feed_image = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+',
+        help_text='This is the image that will be displayed for the case study on the Case Studies list page.'
+    )
+
+    multilingual_field_panels = [
+        ImageChooserPanel('feed_image'),
+        InlinePanel('case_study_documents', label='Case study attachments'),
+    ]
+
+
+class CaseStudyDocument(Orderable):
+    """A model for Case Study documents."""
+
+    page = ParentalKey(CaseStudyPage, related_name='case_study_documents')
+    document = models.ForeignKey(
+        'wagtaildocs.Document',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
+    panels = [
+        DocumentChooserPanel('document'),
+    ]
+
+
+class HistoryDateBlock(StreamBlock):
+    """A block for History event card along the timeline which is shown on the HistoryPage."""
+
+    event_block_editor = StructBlock([
+        ('heading', CharBlock(required=False, max_length=100)),
+        ('description', TextBlock(required=False))
+    ])
+
+
+class HistoryPage(AbstractContentPage):
+    """A model for the History page."""
+
+    subpage_types = []
+
+    timeline_editor = StreamField(HistoryDateBlock, null=True, blank=True)
+
+    translation_fields = AbstractContentPage.translation_fields + ['date_panel']
```

Comment:
Need to update `['date_panel']` to `['translation_fields']`

Reply:
@akmiller01 do you mean `['timeline_editor']` ?